### ISSUE TITLE
BED - Bulk update the old aliases so that they won't collide anymore

### ISSUE THREAD

**Issue by janetgatsby** [patpluspun/fmp_au#73]
The update that fixed the aliases to begin with did not retroactively fix the existing books.  I will test to see if I can bulk update the old aliases so that they won't collide anymore.

**Comment by janetgatsby** [comment]
@patpluspun I don't even remember what this is, do you?

**Comment by patpluspun** [comment]
Yes, it's a retroactive fix for products that create the infinite loops when changing redirects.  I'll run one more test on test.fivemile.com.au, and it will go into today's deployment.

**Comment by patpluspun** [comment]
Update running now.  It's gonna take awhile.

**Comment by patpluspun** [comment]
And the update timed out.  I'm afraid we have too many products for this server to successfully handle all of the bulk updating.  I'll push the change to address all aliases going forward, however.

**Comment by janetgatsby** [comment]
I can move this to the backlog or do you want it in the next milestone?

**Comment by patpluspun** [comment]
I'll take it in the next milestone.  I have affirmed that aliases look good on www, New Releases should not be part of the url anymore when that category is chosen.
